typescript/packages/sphere-viewer/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "@subconsciousnetwork/sphere-viewer",
+  "author": "Subconscious Inc.",
+  "license": "Apache-2.0 OR MIT",
+  "version": "0.1.0",
+  "description": "A demo of Orb.js that renders Noosphere content from any IPFS gateway",
+  "type": "module",
+  "scripts": {
+    "build": "wireit",
+    "build:tsc": "wireit",
+    "clean": "wireit",
+    "serve": "wireit"
+  },
+  "dependencies": {
+    "@reduxjs/toolkit": "^1.9.1",
+    "@subconsciousnetwork/orb": "*",
+    "@web/dev-server": "^0.1.34",
+    "lit": "^2.4.1",
+    "lit-redux-watch": "^0.3.8",
+    "reselect": "^4.1.7",
+    "typescript": "~4.8.2",
+    "vite": "^3.2.3",
+    "vite-plugin-wasm": "^3.1.0",
+    "wireit": "^0.8.0"
+  },
+  "wireit": {
+    "build": {
+      "dependencies": [
+        "build:tsc"
+      ],
+      "command": "vite build"
+    },
+    "build:tsc": {
+      "dependencies": [
+        "../orb:build"
+      ],
+      "command": "tsc"
+    },
+    "serve": {
+      "dependencies": [
+        "build"
+      ],
+      "command": "web-dev-server -r ./dist --open"
```

Comment:
You likely want `"service": true` here, so that you can run `npm run serve --watch`. Without `"service": true`, watch mode won't work because Wireit will be waiting for the `web-dev-server` process to exit, but it never will.

https://github.com/google/wireit#services